lib/motors/can_messages.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include <core/types.hpp>
+#include <io/can.hpp>
+
+namespace hyped::motors {
+
+class VectorControlCanMessages {
```

Comment:
`ControllerCanProcessor`, and implement `ICanProcessor` for this class. Currently, when it receives a message have it respond with an `Error::kInvalidOperation` error message. (this is an increase in scope for the PR, but might as well do it now)